esp_encrypted_img/idf_component.yml

```diff
@@ -1,6 +1,8 @@
-version: "2.5.1"
+version: "2.6.0"
 description: ESP Encrypted Image Abstraction Layer
 url: https://github.com/espressif/idf-extra-components/tree/master/esp_encrypted_img
 dependencies:
   idf:
-    version: ">=4.4"
+    version: ">=5.5"
```

Comment:
This will lock in the component for IDF >= 5.5 release. We should support prior releases and take an exception for newly added feature only